context/app/static/js/components/workspaces/api.ts

```diff
@@ -319,10 +325,8 @@ async function createWorkspaceFetcher(_key: string, { arg: { body, url, headers
   trackEvent({
     category: 'Workspace Creation',
```

Comment:
Should this be 'Workspace Action'?

Reply:
I didn't really take a look at the strings for the existing events. If anything this should be 'Workspaces' and let the `action` differentiate the events.

Reply:
From a grouping perspective, I'm advocating for a consistent category value.  'Workspaces" sounds good to me. I'm seeing three categories currently, 'Workspace Action', 'Workspace Creation', 'Workspace Update'. It may be valuable to have all the Workspace events categorized together, still with `action` differentiating the events.

![image](https://github.com/hubmapconsortium/portal-ui/assets/3586093/60cdde87-3ef5-454a-b03c-5236dd660b3a)

Reply:
Sorry, edited my comment likely while you were responding. I'm in full agreement and can pull those changes into this PR.